pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -32,10 +32,19 @@ func (n *NetworkSpec) IsMultus() bool {
 	return n.Provider == NetworkProviderMultus
 }
 
-// IsHost get whether to use host network provider. This method also preserve
-// compatibility with the old HostNetwork field.
+// IsHost can be used to determine if pods should use host networking by
+// checking whether the cephCluster is configured to use the "host" network provider.
+// This method also maintains compatibility with the old hostNetwork setting
```

Comment:
FWIW, when Travis and I chatted, he agreed that it isn't necessary to have e2e in CI for this since it is such a rare case. Solid local testing should be fine. Also, it isn't necessary to mention that it is supported on a best-effort basis since that is essentially how all FOSS is supported.